sdk/core/core-rest-pipeline/test/defaultLogPolicy.spec.ts

```diff
@@ -0,0 +1,82 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { assert } from "chai";
+import { createPipelineFromOptions } from "../src/createPipelineFromOptions";
+import { PipelinePolicy } from "../src/pipeline";
+import { createPipelineRequest } from "../src/pipelineRequest";
+import { createHttpHeaders } from "../src/httpHeaders";
+import sinon from "sinon";
+import { DEFAULT_RETRY_POLICY_COUNT } from "../src/constants";
+
+describe("defaultLogPolicy", function () {
+  it("should be invoked on every retry", async function () {
+    const request = createPipelineRequest({
+      url: "https://bing.com",
+    });
+
+    let testSignPolicy: PipelinePolicy = {
+      name: "testSignPolicy",
+      sendRequest: async function (request, next) {
+        let response = await next(request);
+        return response;
+      },
+    };
+
+    let pipeline = createPipelineFromOptions({
```

Comment:
style nit: prefer const where possible, surprised the linter isn't yelling at you (though we all know how reliable the linter can be 😉)

Reply:
Yeah I think these are most of the failures in the CI pipeline. You can run locally `rushx lint` to see the errors

Reply:
Thanks for the tip. I might make a git hook for the linter so I don't forget. Fixed!

Reply:
this is 100% why `fix eslint inside of vs code` was on investment backlog - we had some issues in the past with using a git hook